virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc

```diff
@@ -0,0 +1,42 @@
+[id="virt-importing-rhv-vm"]
+= Importing a single Red Hat Virtualization virtual machine
+include::modules/virt-document-attributes.adoc[]
+:context: virt-importing-rhv-vm
+:virt-importing-rhv-vm:
+toc::[]
+
+You can import a single Red Hat Virtualization (RHV) virtual machine into your {product-title} cluster by using the virtual machine wizard or the CLI.
+
+:FeatureName: Importing a Red Hat Virtualization virtual machine into {product-title}
+include::modules/technology-preview.adoc[leveloffset=+1]
+:!FeatureName:
+
+include::modules/virt-importing-vm-prerequisites.adoc[leveloffset=+1]
+
+ifeval::["{VirtVersion}" == "2.4"]
+include::modules/virt-creating-configmap.adoc[leveloffset=+1]
+ifeval::["{VirtVersion}" == "2.4"]
+
+include::modules/virt-importing-vm.adoc[leveloffset=+1]
+
+See xref:virt-importing-rhv-vm-wizard-fields_virt-importing-rhv-vm[virtual machine wizard fields] for more information on the wizard fields.
+
+include::modules/virt-importing-vm-cli.adoc[leveloffset=+1]
+
+
+ifeval::["{VirtVersion}" == "2.4"]
+include::modules/virt-canceling-vm-import.adoc[leveloffset=+1]
+endif::[]
+
+include::modules/virt-troubleshooting-vm-import.adoc[leveloffset=+1]
+
+[id="virt-importing-rhv-vm-wizard-fields_{context}"]
+== Virtual machine wizard fields
```

Comment:
Many of the wizard fields here are either not supported for RHV to CNV VM import, or cannot be set.

Reply:
We already went through the wizard fields that are not supported for RHV to CNV.

@suomiy Can you check whether there are any fields that should not be here?

Reply:
Not necessary. I found the discussion:

<img width="378" alt="image" src="https://user-images.githubusercontent.com/31725355/87914925-20015300-ca7a-11ea-9e95-ac1a8e8aee6f.png">